brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3474 Fix: Forms - Resolved an issue of ReCaptcha field visible after visiting frontend and on refresh it get hide.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3474

### Screenshots

- https://d.pr/v/kE02q5

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Forms.
- Enable ReCaptcha field > Select V3 and Hide reCAPTCHA Badge on frontend
- Visit frontend. Make sure you have added reCaptcha site and secret key in Spectra Admin dashboard > Settings > Integration.
- Make sure reCaptcha field hidden and should not visible on frontend.

### Checklist:
